### DISCUSSION TITLE
Force CPU

### DISCUSSION THREAD

**Discussion by doublex** [pyannote/pyannote-audio#1266] [Q&A] [answered]
Is there a knob to force `VoiceActivityDetection` to run on the CPU (even if cuda available)?
Thanks a lot!

**Comment by hbredin** [comment, marked as answer]
Short answer: not yet.
Long answer: in current `develop` branch, pipelines run on CPU by default and can be sent to a particular device using `pipeline.to(torch.device("cuda"))`.

**Comment by doublex** [comment]
@hbredin 
Thanks a lot!
